### PR TITLE
Vertical lists - Top-border variant

### LINKED ISSUE #148: Enhancement - Provide ability to remove border-top from vertical lists
The vertical lists should have a `border-top` when used for news items, however other vertical lists in the design should not have `border-top`. It would be great to be able to easily specify whether the component should or shouldn't have a `border-top`.

Example can be seen on the Agency Landing page 

**Current**

before-listing

**Desired**

after-listing

### PR DESCRIPTION
**Pull request by TrebBrennan**
Fixes #148 
Adds classes to accommodate vert list rows without border top.

### CODE REVIEW

**Comment by klepas** [comment]
Only cursory.

Generally speaking, the fix seems to work, though I’m not overly in favour of adding a class that only does presentational things.

**Comment by TrebBrennan** [comment]
We really just need it to be simple for people to use and have control over; so I don't mind what classes we use :)

**Comment by klepas** [comment]
> @klepas I understand where you're coming from. In this case we've already added a modifier to .list-vertical to describe the three and four column views. Should we treat border variants as a modifier, or should have another set of classes to describe decoration? In this case the latter makes more practical sense to me.

Practically speaking, agreed. Philosophically speaking and in relation to our conventions, I want to avoid non-semantic/presentational markup as much as we practically can; it’s going to be a compromise I think.

tl;dr: LGTM, but another one of those things I want to add to the list of stuff to review at some point — in that respect it adds to our tech debt.

Feel free to merge. (:
